### PR TITLE
Add limit parameter on Search

### LINKED ISSUE #746: Add limit parameter on Search API
**Is your feature request related to a problem? Please describe.**

From PR https://github.com/kiwicom/margarita/pull/735 , some performance problems arise when loading way too many results on screen and the body is scrollable (e.g on resizing the windows) on web version with mobile resolution.

**Describe the solution you'd like**

For solving it, we should consider implementing a sensible `limit` parameter on the search API call, for example, 50 results.

cc  @JoseDuSV

### PR DESCRIPTION
**Pull request by aalises**
Added limit parameter on search with a value of `50`. Closes #746

### CODE REVIEW

**Comment by FilipMessa** [comment]
What is the motivation behind this?

**Comment by aalises** [comment]
@FilipMessa So now on web version, when on a mobile resolution, the whole body is scrollable. That creates performance issues on resizing, for example, as we are re-rendering 200 cards. This is a *temporary* solution as discussed with @JoseDuSV for optimization purposes. Ideally, when pagination is implemented in Tequila, we should implement it with pagination with a load more button for example.

File: apps/core/scenes/search/SearchContext.js [hunk 1]
```diff
@@ -89,7 +91,7 @@ type State = {|
 const defaultDepartureDate = DateFNS.addDays(new Date(), 1);
 const defaultReturnDate = DateFNS.addDays(defaultDepartureDate, 2);
 const defaultSortBy = 'QUALITY';
-
+const defaultLimit = 50;
```
Comment (jvaclavik) [inline apps/core/scenes/search/SearchContext.js]: Maybe we should move default values to `margarita-config`, what do you think?

Comment (aalises) [reply apps/core/scenes/search/SearchContext.js]: @jvaclavik done!

**Pull request merged**